build-support/bin/ci.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```

Comment:
I don't love putting this in build-support/bin, we already have a bin directory, can we use that?